### PR TITLE
[instant] Add icons to token selector

### PR DESCRIPTION
**Pull request by BMillman19**
## Description

* Add a new `assets/icons` folder with token icons
* Update the webpack config to bundle images

## Testing instructions

http://0x-instant-dogfood.s3-website-us-east-1.amazonaws.com/

## Types of changes

* New feature (non-breaking change which adds functionality)

## Checklist:

*   [ ] Prefix PR title with `[WIP]` if necessary.
*   [ ] Prefix PR title with bracketed package name(s) corresponding to the changed package(s). For example: `[sol-cov] Fixed bug`.
*   [ ] Add tests to cover changes as needed.
*   [ ] Update documentation as needed.
*   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/instant/src/components/erc20_token_selector.tsx [hunk 1]
```diff
@@ -95,9 +96,13 @@ class TokenSelectorRow extends React.Component<TokenSelectorRowProps> {
                         <Container marginRight="10px">
                             <Circle diameter={30} rawColor={token.metaData.primaryColor}>
                                 <Flex height="100%">
-                                    <Text fontColor={ColorOption.white} fontSize="8px">
-                                        {displaySymbol}
-                                    </Text>
+                                    {!_.isUndefined(iconUrlIfExists) ? (
```
[the remark this answers is not in the document]
Comment (BMillman19) [reply packages/instant/src/components/erc20_token_selector.tsx]: I understand where your coming from but this is the convention we use all over the codebase and I'd rather be consistent

File: packages/instant/src/components/erc20_token_selector.tsx [hunk 2]
```diff
@@ -79,6 +79,7 @@ class TokenSelectorRow extends React.Component<TokenSelectorRowProps> {
     public render(): React.ReactNode {
         const { token } = this.props;
         const displaySymbol = assetUtils.bestNameForAsset(token);
+        const TokenIcon = require(`../assets/icons/${token.metaData.symbol}.svg`);
```
Comment (fragosti) [inline packages/instant/src/components/erc20_token_selector.tsx]: apparently `import()` is a thing? Does that work?

Also this doesn't allow people to pass icons in which is kind of a bummer. Any way we can get that functionality?

**Comment by BMillman19** [comment]
@fragosti the icons are converted into react components and bundled into the main bundle, they are all present with the first load: https://github.com/jhamlet/svg-react-loader

File: packages/instant/src/components/erc20_token_selector.tsx (hunk 2 above)
Comment (BMillman19) [reply packages/instant/src/components/erc20_token_selector.tsx]: hmm, it works with `require` but not `import`

Comment (fragosti) [reply packages/instant/src/components/erc20_token_selector.tsx]: @BMillman19 what you said makes sense about `iconUrl`

**Pull request merged**